### DISCUSSION TITLE
Fejl ved frivillige fra venteliste

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#29] [Bugs] [answered]
Fra Malene: 

Jeg skulle slå en frivillig (Lisa Hansen) op, for at se hvornår hun havde næste vagt. Da jeg gjorde det undrede det mig at jeg ikke kunne se hendes vagt i dag, selvom hun står på vagtplanen. Hun sagde at hun først har stået på venteliste og sidenhen fået invitationen til vagten.

Så der må været et glip et sted.

Vagten hun er på:
![image](https://github.com/xooom-community/product/assets/167326849/5aa82fe9-f03e-40ac-a6f7-fdea5cbab975)

Når jeg slår hende op:
![image](https://github.com/xooom-community/product/assets/167326849/66cb5a12-8be1-4e74-a22c-cd3789d6d4dd)

Det samme gælder for frivillige/medarbejdere der er tildelt en vagt via backenden. Deres vagter fremgår heller ikke af denne oversigt.

**Comment by henrikbak** [comment, marked as answer]
Oprettet i development: https://github.com/orgs/xooom-community/projects/1/views/1?pane=issue&itemId=67299775
